source/plugins/cloudstack-kubernetes-service.rst

```diff
@@ -329,6 +329,22 @@ Token for dashboard login can be retrieved using the following command:
    # kubectl --kubeconfig /custom/path/kube.config describe secret $(kubectl --kubeconfig /custom/path/kube.config get secrets -n kubernetes-dashboard | grep kubernetes-dashboard-token | awk '{print $1}') -n kubernetes-dashboard
 
 
+Kubernetes compatibility Matrix
+#################################
+
++--------------+---------------------------------+-----------------------------+-------------+
+|ACS Version   |  Supported Kubernetes Versions  |  K8s Node Template          |  User       |
++==============+=================================+=============================+=============+
+| 4.14.x       | v1.11 onward (< 1.18)           | CoreOS                      | core        |
```

Comment:
Instead of user, should we say "SSH User" or "Login User"? and simply say "CKS Template"